CHANGELOG

```diff
@@ -4,7 +4,7 @@ Qtile x.xx.x, released XXXX-XX-XX:
       - cairocffi must be upgraded to >= 1.6.0
     !!! Notice for packagers - wlroots (optional dependency) bump !!!
       - Qtile's wayland backend now requires on wlroots 0.16 (and pywlroots 0.16)
-    !!! config breakage/changes !!!
+    !!! config breakage/changes - use `qtile migrate` to update your config !!!
```

Comment:
I think this needs a rebase now (though we don't have any breaking config changes yet for the next release, but I assume there will be some :).